### ISSUE TITLE
Accounts bug - duplicate character entries

### ISSUE THREAD

**Issue by Gogk** [torilmud/issues#217]
I accidentally used the normal log with a character not my regular log in. It created a duplicate of one of my characters. Then another somehow. So now I have three of one character and I can't remove the extras.

**Comment by Inames** [comment]
Lol, that's weird. I'm surprised that hasn't been discovered before... particularly when everyone was getting used to the new account system...

**Comment by Inames** [comment]
So to clarify, as that is kind of vague. You accidentally tried to log in with one of the characters on your list that wasn't the primary account name. And that allowed you to log into your account. But it added a duplicate of the character's name to your list?

**Comment by Gogk** [comment]
Correct. I was logged out of the game, I went to log back in while I was reordering my characters and I think I accidentally logged out and for some reason my brain said "Assicellon" Go ahead type that in George... And just out of habit I put my pword in and it logged me into the game. I logged out. Went to log back in using the normal log in. When I did, I noticed two assicellons. And for some reason now there's a third...

I'm really good at breaking things lol
